py/surveyqa/nightly.py

```diff
@@ -160,7 +161,7 @@ def get_skypathplot(exposures, tiles, width=600, height=300):
     
     #- Plot options
     night_name = exposures['NIGHT'][0]
-    string_date = night_name[:4] + "-" + night_name[4:6] + "-" + night_name[6:]
+    string_date = night_name[4:6] + "-" + night_name[6:] + "-" + night_name[:4]
```

Comment:
I'm going to switch this back to what you originally had (YEAR-MM-DD) instead of MM-DD-YEAR, since everywhere else in DESI we refer to the "night" as YEARMMDD and chose that order so that sorting by "night" will be sorted in time.  In this case we aren't sorting by plot title but I'd like to keep the order of YEAR-MM-DD consistent throughout our code and plots.  Otherwise this looks good.

Reply:
Note to self: we might just remove the date from this title since it is redundant with the header text that also includes the night.  If the goal of adding dashes etc. is to make it more human friendly, we might consider going all the way to "Thurs Dec 12 2019" or something like that that is very obviously a date and not a bunch of numbers.  Not for this PR though.